pyomo/common/fileutils.py

```diff
@@ -8,10 +8,28 @@
 #  This software is distributed under the 3-clause BSD License.
 #  ___________________________________________________________________________
 
+import inspect
 import os
+import platform
 import six
 
 
+def thisFile():
```

Comment:
Why camelCase?

Reply:
Good question.  Random choice? I have removed it.